src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -209,6 +209,14 @@ public class NeoForgeMod {
      */
     public static final Holder<Attribute> BLOCK_REACH = ATTRIBUTES.register("block_reach", () -> new RangedAttribute("neoforge.block_reach", 4.5D, 0.0D, 1024.0D).setSyncable(true));
 
+    /**
+     * Grants the player the ability to use creative flight when not in creative mode.
+     * Anything above zero allows flight.
+     *
+     * @see net.neoforged.neoforge.common.extensions.IPlayerExtension#mayFly
+     */
+    public static final Holder<Attribute> CREATIVE_FLIGHT = ATTRIBUTES.register("creative_flight", () -> new RangedAttribute("neoforge.creative_flight", 0D, 0D, Double.MAX_VALUE).setSyncable(true));
```

Comment:
The removal modifier should be a -1 multiply total operation

Reply:
It may be worth using part of my BooleanAttribute documentation here to establish standard modifier values https://github.com/Shadows-of-Fire/Apothic-Attributes/blob/1.20/src/main/java/dev/shadowsoffire/attributeslib/impl/BooleanAttribute.java#L14-L25

Reply:
What happens when there are 2 removals? Does the clamp happen between modifiers?

Reply:
Copypasting external response:

Vanilla code for multiply total is 
```
for(AttributeModifier attributemodifier2 : this.getModifiersOrEmpty(AttributeModifier.Operation.MULTIPLY_TOTAL)) {
   d1 *= 1.0D + attributemodifier2.getAmount();
}
```

so if you apply a -1 you get d1 *= 0 and nothing else can do anything (since only multiplications are left)